lib/net/http.rb

```diff
@@ -736,6 +737,18 @@ def set_debug_output(output)
     # it raises a Net::ReadTimeout exception. The default value is 60 seconds.
     attr_reader :read_timeout
 
+    # Number of max retries in case of Net::ReadTimeout, IOError, EOFError,
```

Comment:
How about "Maximum number of times to retry an idempotent request in case of …"

Both a slight re-phrasing plus the addition of "idempotent request" so we're clear on which types of requests get retried.